force-app/main/default/classes/SOQL.cls

```diff
@@ -117,7 +152,9 @@ public with sharing class SOQL {
      * @return A list of Case records.
      */
     public static List<Case> getFilterCaseOrigins() {
-        return null; // Replace null with the variable you used to store the result
+        List<Case> cases = [SELECT Id, Origin FROM Case WHERE Origin NOT IN ('Web', 'Email', '')];
```

Comment:
This is a pretty small set of inputs, but I also like to put them in a Set or List and then use that as a bind variable in the query.